opentelemetry-zio-logging/src/main/scala/zio/telemetry/opentelemetry/zio/logging/ZioLogging.scala

```diff
@@ -0,0 +1,60 @@
+package zio.telemetry.opentelemetry.zio.logging
+
+import io.opentelemetry.api.trace.{Span, SpanContext}
+import io.opentelemetry.context.Context
+import zio._
+import zio.logging.LogFormat
+import zio.logging.LogFormat.label
+import zio.telemetry.opentelemetry.context.ContextStorage
+
+trait ZioLogging {
+
+  /**
+   * Will print traceId from current span or nothing when not in span
+   */
+  def traceId(): LogFormat
+
+  /**
+   * Will print spanId from current span or nothing when not in span
+   */
+  def spanId(): LogFormat
+
+  /**
+   * Label with `traceId` key and [[traceId]] value
+   */
+  def traceIdLabel(): LogFormat = label("traceId", traceId())
+
+  /**
+   * Label with `spanId` key and [[spanId]] value
+   */
+  def spanIdLabel(): LogFormat = label("spanId", spanId())
+}
+
+object ZioLogging {
+
+  def live: ZLayer[ContextStorage, Nothing, ZioLogging] = ZLayer {
```

Comment:
It should be named `logFormats` and return `ZLayer[ContextStorage, Nothing, LogFormats]` instead.

Reply:
Fixed